contents/docs/cdp/build/index.md

```diff
@@ -1,40 +1,40 @@
 ---
-title: Build a data connector
+title: Build a transformation or destination
 sidebar: Docs
 showTitle: true
 ---
 
-Connectors can add more information to an event, modify existing properties, import or export data, or trigger a range of other activities.
+Transformation and destination can add more information to an event, modify existing properties, import or export data, or trigger a range of other activities.
 
-Before building your first connector it's important to understand how data flows through connectors in the first place. There are two critical concepts to remember:
+Before building your first one it's important to understand how data flows through them in the first place. There are two critical concepts to remember:
 
-1. Connectors usually act on _single events_ coming in to PostHog.
+1. Transformation and destination usually act on _single events_ coming in to PostHog.
 
-2. The output of one connector will go into the next connector, creating a chain.
+2. The output of one goes into the next, creating a chain.
```

Comment:
Lets break this up into two: 
Transformation = can add more information to an event, modify existing properties
destination = export data
not possible = import data or trigger a range of other activities

They both act on a single event. Transformations work in order creating a chain.
Plugin Destinations operate at the end of the pipeline parallel with each other and writing to CH